x-pack/legacy/plugins/reporting/server/lib/jobs_query.ts

```diff
@@ -34,12 +35,13 @@ interface CountAggResult {
   count: number;
 }
 
-export function jobsQueryFactory(server: ServerFacade) {
+export function jobsQueryFactory(
+  server: ServerFacade,
+  elasticsearch: ReportingSetupDeps['elasticsearch']
+) {
   const index = server.config().get('xpack.reporting.index');
   // @ts-ignore `errors` does not exist on type Cluster
-  const { callWithInternalUser, errors: esErrors } = server.plugins.elasticsearch.getCluster(
-    'admin'
-  );
+  const { callWithInternalUser, errors: esErrors } = elasticsearch.getCluster('admin');
```

Comment:
I'm pretty sure the `esErrors` reference is broken in this PR. Not sure what the solution is for getting the map of ES errors that we had in Legacy.